styles.css

```diff
@@ -7,10 +7,178 @@ body {
   overflow-x: hidden;
 }
 
+.background {
+  overflow-x: hidden;
+}
+
 a {
   text-decoration: none;
 }
 
+.error {
+  background-color: white;
+  color: red;
+}
+
+.blur {
+  filter: blur(8px);
+}
+
+.introbox {
+  overflow: hidden;
+}
+
+header {
+  background-color: white;
+  box-sizing: border-box;
+  position: fixed;
+  width: 100%;
+  z-index: 4;
+}
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-menu {
+  display: flex;
+  justify-self: end;
+  align-items: center;
+  gap: 20px;
+}
+
+.nav-brand {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.nav-link {
+  color: #344563;
+  transition: 0.7s ease;
+}
+
+.nav-link:hover {
+  color: #6070ff;
+}
+
+.hamburguer {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+.popup .box2 {
+  top: 10px;
+  height: 100%;
+  width: 100%;
+}
+
+.popup .box2 h1 {
+  padding: 0;
+  margin: 0;
+}
+
+.popup .box2 img {
+  max-width: 95%;
+}
+
+.popup .box1 {
+  display: flex;
+  justify-content: space-between;
+  width: 95%;
+}
```

Comment:
- [ ] It would be better to style the line as in the Figma design in order to match it 
- Yours 
<img width="351" alt="Screenshot 2023-04-11 at 22 19 34" src="https://user-images.githubusercontent.com/61790587/231301734-de10d206-afe2-4396-baf2-5936a5f4b7e5.png">

- Design 
<img width="320" alt="Screenshot 2023-04-11 at 22 19 41" src="https://user-images.githubusercontent.com/61790587/231301713-abfd0834-5f1c-4604-b859-2e1a3520fe70.png">